libcloud/common/dimensiondata.py

```diff
@@ -355,11 +472,14 @@ def __repr__(self):
 
 
 class DimensionDataVIPNode(object):
-    def __init__(self, id, name, status, ip):
+    def __init__(self, id, name, status, ip, connection_limit='10000',
```

Comment:
It looks like the limit values are string and not integers? Or just the default values are strings.

In any case, would be better if they were integers.